### PR TITLE
Update to v23.7.3

### PR DESCRIPTION
**Pull request by chris-gds**
Inclues:
 - Fix to resolve input zoom-text overlap PR #1793

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,10 +7,11 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
-## 23.7.2
-* Fix Brexit countdown in the contextual sidebar to use appropriate lang and text direction ([PR #1790](https://github.com/alphagov/govuk_publishing_components/pull/1790))
+## 23.7.3
 * Fix to resolve input zoom-text overlap ([PR #1793](https://github.com/alphagov/govuk_publishing_components/pull/1793))
 
+## 23.7.2
+* Fix Brexit countdown in the contextual sidebar to use appropriate lang and text direction ([PR #1790](https://github.com/alphagov/govuk_publishing_components/pull/1790))
```
Comment (injms) [inline CHANGELOG.md:14]: Small edit - needs an extra newline:
```suggestion
* Fix Brexit countdown in the contextual sidebar to use appropriate lang and text direction ([PR #1790](https://github.com/alphagov/govuk_publishing_components/pull/1790))

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,10 +7,11 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
-## 23.7.2
-* Fix Brexit countdown in the contextual sidebar to use appropriate lang and text direction ([PR #1790](https://github.com/alphagov/govuk_publishing_components/pull/1790))
+## 23.7.3
```
Comment (injms) [inline CHANGELOG.md:10]:
```suggestion
## 23.7.3

```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,10 +7,11 @@
...
 * Fix to resolve input zoom-text overlap ([PR #1793](https://github.com/alphagov/govuk_publishing_components/pull/1793))
 
+## 23.7.2
```
Comment (injms) [inline CHANGELOG.md:13]:
```suggestion
## 23.7.2

```

Comment (chris-gds) [reply CHANGELOG.md:14]: 🙈  spacing added ✅

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,10 +7,14 @@
...
-## 23.7.2
-* Fix Brexit countdown in the contextual sidebar to use appropriate lang and text direction ([PR #1790](https://github.com/alphagov/govuk_publishing_components/pull/1790))
+## 23.7.3
+
 * Fix to resolve input zoom-text overlap ([PR #1793](https://github.com/alphagov/govuk_publishing_components/pull/1793))
 
+## 23.7.2
+
+* Fix Brexit countdown in the contextual sidebar to use appropriate lang and text direction ([PR #1790](https://github.com/alphagov/govuk_publishing_components/pull/1790))
+*
```
Comment (injms) [inline CHANGELOG.md:17]:
```suggestion

```

**Review by injms — APPROVED** [review]
💎👍

**Pull request merged**